scipy/stats/_stats_py.py

```diff
@@ -9336,6 +9336,26 @@ def friedmanchisquare(*samples):
     ----------
     .. [1] https://en.wikipedia.org/wiki/Friedman_test
 
+    Examples
+    --------
+    Suppose we want to test that the average reaction time of three different
+    drugs is different. We can use the friedman test for this analysis by
+    measuring the reaction time of 10 patients on each of the three drugs.
+    We set the null hypothesis to be the claim that the average reaction
+    time of the three different drugs is approximately the same.
+
+    >>> from scipy.stats import friedmanchisquare
+    >>> drug1 = [4, 6, 3, 4, 3, 2, 2, 7, 6, 5]
+    >>> drug2 = [5, 6, 8, 7, 7, 8, 4, 6, 4, 5]
+    >>> drug3 = [2, 4, 4, 3, 2, 2, 1, 4, 3, 2]
+    >>> friedmanchisquare(drug1, drug2, drug3)
+    FriedmanchisquareResult(statistic=13.351351351351344, pvalue=0.0012612201221243592)
+
+    According to the results the test statistic is about 13.35135 with a p
+    value of 0.00126. Since the p value is less than 0.05 (value of alpha)
+    we can reject the null hypothesis and conclude that the average response
+    time of the three drugs is different.
```

Comment:
I suggest having a look at other examples to soften a bit the wording. Also it's better to not reference exact values as they might change (multiple reasons.)

Reply:
This is the example I consulted https://www.statology.org/friedman-test-python/. Should I put this as a reference?

Reply:
Can you try to find out where the example is coming from? There might be a paper (maybe the original paper about the test) or another reference we could use. That would be better as referencing a website which has no source.

Otherwise, I think it would be better to find another example.

Reply:
If you're a student you might have access through your institution. Otherwise you find most papers in PDF distributed on things like researchgate or on authors' websites